source/documentation/metrics-install-guide/Upgrading_Metrics_Store.html.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Replacing Fluentd with Rsyslog
+---
+
+# Chapter 5: Upgrading Metrics from 4.2
+
+in oVirt 4.3 Fluentd was replaced  by Rsyslog to gather metrics and logs from the oVirt hosts and engine.
+
+Rsyslog is used to collect and parse Engine, VDSM logs and Collectd metrics, add additional metadata and send the data to the remote metrics store.
+
+Note: this does not upgrade the Metrics store but the collectors.
+
+## pre-req
+
+* oVirt Metrics 4.3 with working metrics OpenShift deployment.
+* oVirt host 4.3 which was previously upgraded from 4.2.
+
+
+
+## Procedure
+
+1. Log in to the upgraded Ovirt Host machine using SSH.
+2. Copy `config.yml.example` to create `config.yml`:
+
+```
+# cp /etc/ovirt-engine-metrics/config.yml.example /etc/ovirt-engine-metrics/config.yml.d/config.yml
+```
+
+3. Edit the `ovirt_env_name` and `elasticsearch_host` parameters in config.yml and save the file. These parameters are mandatory and are documented in the file.
+4. Deploy Collectd and Rsyslog on the hosts:
```

Comment:
This is the only step that is mandatory.
@eslutsky We can also suggest to run it weekly by cron job.

Reply:
how do you suggest write this ?